### DISCUSSION TITLE
How to load oauth token to http post request?

### DISCUSSION THREAD

**Discussion by NSH531** [curl/curl#8774] [Q&A]
Thanks
Netanel
netanel.cyber@gmail.com
nsh531@gmail.com

**Comment by smaeda-ks** [comment]
@NSH531 You should add more context/details as it might depend on the API provider you're talking about. Generally speaking, if you already obtained your access token, you can add that to the `Authorization` request header using `-H` option.

```
curl -H 'Authorization: Bearer your_access_token' -XPOST https://api.example.com/foo
```

https://curl.se/docs/manpage.html#-H

**Reply by NSH531** [reply]
how to do this in python or php???
